tsconfig.json

```diff
@@ -13,12 +11,8 @@
     // "disableSolutionSearching": true,                 /* Opt a project out of multi-project reference checking when editing. */
     // "disableReferencedProjectLoad": true,             /* Reduce the number of projects loaded automatically by TypeScript. */
 
-    "lib": ["ES2022"],
-    "moduleResolution": "node",
-    "target": "ES2022",
-    "esModuleInterop": true,
-
     /* Language and Environment */
+    "target": "es2016",                                  /* Set the JavaScript language version for emitted JavaScript and include compatible library declarations. */
```

Comment:
Can we bump this into at least ES2022? This won't run in the browser, but in an environment we control...